x-pack/plugin/core/src/main/java/org/elasticsearch/protocol/xpack/license/PostStartBasicResponse.java

```diff
@@ -0,0 +1,237 @@
+/*
```

Comment:
With the new way we are not sharing request/response, there is no reason to touch these, nor more them into protocol. These PRs now should be touching nothing in `x-pack` or `server`.